fluent-bit-image/v1.6/Dockerfile.windows

```diff
@@ -0,0 +1,69 @@
+# escape=`
+
+ARG WINDOWS_VERSION=ltsc2019
+
+#
+# Builder Image - Windows Server Core
+#
+FROM mcr.microsoft.com/windows/servercore:$WINDOWS_VERSION as builder
+
+# The FLUENTBIT_VERSION ARG must be after the FROM instruction
+ARG FLUENTBIT_VERSION=1.7.0

```

Comment:
@ahma should this directory be called `v1.7` since the Dockerfile uses Fluentbit 1.7?